core/openapi.yaml

```diff
@@ -343,58 +343,41 @@ components:
           type: number
           format: double
           example: 0.25
-        timetable_gamma:
+        gamma:
           type: number
           format: double
           example: 0.5
-        tractive_effort_curve:
```

Comment:
while it is true that this field is not there anymore, it was replaced by `effort_curves`, which should be added